example/src/androidTest/kotlin/com/example/afterpay/DisableAnimationsRule.kt

```diff
@@ -0,0 +1,38 @@
+package com.afterpay.android
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import java.io.IOException
+
+private const val SHELL_COMMAND = "settings put global %s %d"
+private const val TRANSITION_ANIMATION_SCALE = "transition_animation_scale"
+private const val WINDOW_ANIMATION_SCALE = "window_animation_sc"
+private const val ANIMATOR_DURATION = "animator_duration_scale"
+
+class DisableAnimationsRule : TestRule {
```

Comment:
Ah I should do something like this on iOS too.

Reply:
I think I assumed you'd get it for free, given the generally better quality of the testing apparatus on iOS.

Reply:
I still need to call `UIView.setAnimationsEnabled(false)` or something.